addons/website/controllers/backend.py

```diff
@@ -55,3 +55,28 @@ def check_create_access_rights(self, models):
             model: request.env[model].check_access_rights('create', raise_exception=False)
```

Comment:
Answering a few points (not all) before I jump back into the commit and check the rest:
- The dark mode is actually only available on enterprise, so that purple is the enterprise one.
- As I said in another comment, the tips are more a sign of "please be patient" than anything else. So not a big deal if they don't all appear. :)
- To test on local, you should launch the server with the `--workers` flag (I put 4, I guess any number could work). Otherwise, it cannot install the modules and track their installation in parallel. ;)

Reply:
> And when the real one then starts (after having configured everything) both "interval" seem to exist at the same time (even if the tips displayed seem correct) open_mouth

Do you mean both `tipsInterval`s? The first one is cleared when the `useEffect` is called again, with `return () => clearInterval(tipsInterval);`.

Reply:
> The dark mode is actually only available on enterprise

Ah yes that's right haha
Anyway if PDE is okay with this color, then it's okay for me too :wink: 
 
 > To test on local, you should launch the server with the --workers flag
 
 Thanks for the tip :smile: 
 
 > Do you mean both tipsIntervals? The first one is cleared when the useEffect is called again, with return () => clearInterval(tipsInterval);.
 
 My bad, I don't know how I saw that but the issue I mentioned does in fact not happen :no_mouth:

Reply:
> On the first page of the configurator, so the loading hasn't even started yet, the useEffect applied on selectedFeatures is triggered and so the waitingTips interval function is triggered too, browsing the tips array for "nothing"

With the addition of a `showWaitingMessages` state/prop, this should be good now and only apply the effects if it is specifically requested with the `showLoader()` function.